Resources/Prototypes/Reagents/gases.yml

```diff
@@ -190,7 +190,7 @@
   desc: reagent-desc-nitrogen
   physicalDesc: reagent-physical-desc-gaseous
   flavor: bitter
-  color: "#808080"
+  color: "#a1e1ff"
```

Comment:
I think red like the nitrogen tanks/canisters would make more sense than blue?

Reply:
Yeah I thought of that but I just felt like red was a weird color to give to _nitrogen_. Something that makes up most of air not being blue just feels wrong.